Apps/Src/Print_Queue.c

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */
+/** Print_Queue.c
+ * Queue that holds all characters to be printed that Task_Print needs to send to UART.
+ */
+
+#include "Print_Queue.h"
+#include "RTOS_BPS.h"
+#include "stdio.h"
+#include <stdarg.h>
+#include "BSP_UART.h"
+
+
+#define FIFO_TYPE char
+#define FIFO_SIZE (257)
+#define FIFO_NAME Print_Fifo
+#include "fifo.h"
+
+static Print_Fifo_t printFifo;
+static OS_SEM printFifo_ready;
+static OS_MUTEX printCall_Mutex;
+
+/**
+ * @brief Initializes the print queue
+ * @param none
+ * @return none
+ */
+void Print_Queue_Init() {
+    Print_Fifo_renew(&printFifo);
+    RTOS_BPS_SemCreate(&printFifo_ready, "readyPrint", 0);
+    RTOS_BPS_MutexCreate(&printCall_Mutex, "printCall mutex");
+}
+
+/**
+ * @brief Initializes the print queue
+ * @param buffer String of formatted text to be added to the buffer
+ * @return If the write was successful (there was room in the buffer)
+ */
+
+
+bool Print_Queue_Append(char *buffer) {
```

Comment:
this never returns anything but true. when does it return false?

Reply:
no longer a bool